dts/bindings/gpio/gpio-controller.yaml

```diff
@@ -15,4 +15,27 @@ properties:
     ngpios:
       type: int
       required: false
-      description: Number of gpios supported
+      description: |
```

Comment:
The concept of slot doesn't seem to be used anywhere else in the documentation, maybe we should stick to the terminology that is used? We could also shorten it:

_The number of GPIO lines in use. This property allows GPIO driver to determine how many physical GPIO lines are supported by the given hardware module. As an example: setting "ngpios = <18>;" informs the driver that only the first 18 GPIOs, at local offset 0 .. 17, are in use. For cases in which there might be holes in the slot range, this value should be the max GPIO line number - 1._